platform/windows/export/export_plugin.cpp

```diff
@@ -386,21 +386,26 @@ Error EditorExportPlatformWindows::_rcedit_add_data(const Ref<EditorExportPreset
 		args.push_back("--set-product-version");
 		args.push_back(product_version);
 	}
+	//Having a standard naming fallback preventing showing up as Godot Engine
+	args.push_back("--set-version-string");
+	args.push_back("ProductName");
+	if (product_name.is_empty())
+		args.push_back(GLOBAL_GET("application/config/name"));
+	else
+		args.push_back(product_name);
```

Comment:
`GLOBAL_GET("application/config/name")` can be empty, so should be checked as well. Also, Godot code style require {} around all if blocks, use can use `clang-format` no automatically fix code style - see [Code style guidelines](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#c-and-objective-c).

```suggestion
	if (product_name.is_empty()) {
		String app_name = GLOBAL_GET("application/config/name");
		if (app_name.is_empty()) {
			args.push_back("Unnamed"); //Note: different platforms use diffrent names in this case: "Unnamed", "Godot Game".
		} else {
			args.push_back(app_name);
		}
	} else {
		args.push_back(product_name);
	}
```

Reply:
there is stuff missing like making this for file description wich shows up in task manager so i'll implement it myself

Reply:
does it look good to you now?